packages/editor/src/components/widgets/input/IconInput.tsx

```diff
@@ -3,17 +3,30 @@ import type { IvyIcons } from '@axonivy/editor-icons';
 import IvyIcon from '../IvyIcon';
 import type { InputProps } from './Input';
 import Input from './Input';
-import { forwardRef } from 'react';
+import { forwardRef, useEffect, useRef } from 'react';
 
 export type IconInputProps = InputProps & {
   icon: IvyIcons;
+  initFocus?: boolean;
 };
 
-const IconInput = forwardRef<HTMLInputElement, IconInputProps>(({ icon, ...props }, forwardedRef) => {
+const IconInput = forwardRef<HTMLInputElement, IconInputProps>(({ icon, initFocus, ...props }, forwardedRef) => {
```

Comment:
is this iconInput used somewhere with a ref? because if not you can also remove this forwardRef stuff.